code/script.js

```diff
@@ -0,0 +1,111 @@
+const username = "Mimmisen";
+const API_URL = `https://api.github.com/users/${username}`;
+const REPOS_URL = `https://api.github.com/users/${username}/repos`;
+
+// function for token
+const options = {
+  method: "GET",
+  headers: {
+    Authorization: `token ${API_TOKEN}`,
+  },
+};
+//----------------------------------FETCH 1-------------------------------------
+// getting profile pic,user name and link to my github
+fetch(API_URL, options)
+  .then((res) => res.json())
+  .then((data) => {
+    headerBox.innerHTML += `
+    <div>
+    <a href="${API_URL}">
+    <img src="${data.avatar_url}" alt="Avatar of ${data.login}">
+    </a>
+    </div>
+    <a href="https://github.com/Mimmisen"><h1 class="username">${data.name}<h1></a>
+
+    `;
+  });
+
+//----------------------------------FETCH 2-------------------------------------
+fetch(REPOS_URL, options)
+  .then((res) => res.json())
+  .then((data) => {
+    //To get my projects and filter them so only the ones starting with projects are displayed
+    const technigoProjects = data.filter(
+      (repo) => repo.name.startsWith("project-") && repo.fork
+    );
+
+    //Count the repos and show the chart
+    showChart(technigoProjects.length);
+
+    // repo + link to pull request and date for latest update
+    technigoProjects.forEach((repo) => {
+      //Get the name of the repo
+      const reponame = repo.name;
+
+      //declaring reponame to the function to display amount of push I have done for each project
+      fetchTechnigo(reponame);
+      //get the url fo each repo
+      const projectUrl = repo.html_url;
+      //get the latest push/update of the repo
+      const latestPushRepo = new Date(repo.updated_at).toLocaleDateString(
+        "sv-SE",
+        {
+          year: "numeric",
+          month: "long",
+          day: "numeric",
+        }
+      );
+
+      //displaying project url, name and latest push
+      document.getElementById("section-projects").innerHTML += `
+      <div class="section-projects__card box-style">
+      <div>
+     <h3 class="repo-title bold-text"><a class="nav-link" href='${projectUrl}'>${reponame}</a></h3>
+     <p><span class="bold-text">Latest update:</span> ${latestPushRepo}</p>
+     <p id="commits-${reponame}"></p>
+     </div>
+     </div>
+     `;
+    });
```

Comment:
here you are missing the names of the main branches (also minimum requirements), so you could add one line more to your innerHTML (if I remember correctly it was repo.default_branch how you get the names of the main branch)

Reply:
Yes and now I have added it!